fecgov/openFEC
# Update index idx_sched_a_cmte_id_dt_sub_id to use desc order

## Summary (required)
- Resolves #4352 

## How to test the changes locally
Download the branch 
- run the migration file
- (optional) run pytest

Make sure migration file run successfully.  
Connect to local cfdm_test database and run the following query to make sure all 23 partition has this new index.

select tablename, count(*)
from pg_indexes 
where tablename like 'fec_fitem_sched_a%'
and indexname like 'idx_sched_a_%_cmte_id_dt_sub_id_desc'
group by tablename
order by tablename;


## Impacted areas of the application
List general components of the application that this PR will affect:

-  None



## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()
